eth2deposit/intl/ar/cli/generate_keys.json

```
@@ -0,0 +1,35 @@
+{
+    "validate_password": {
+        "msg_password_prompt": "اكتب كلمة المرور التي تؤمن متجر (متاجر) مفاتيح المدقق",
+        "msg_password_confirm": "كرر للتأكيد",
+        "err_password_mismatch": "خطأ: القيمتان اللتان تم إدخالهما غير متطابقتين. الرجاء الكتابة مرة أخرى."
+    },
+    "generate_keys_arguments_decorator": {
+        "num_validators": {
+            "argument": "--num_validators",
+            "help": "عدد مفاتيح المدققين التي تريد إنشاؤها (يمكنك دائمًا إنشاء المزيد في وقت لاحق)",
+            "prompt": "الرجاء اختيار عدد المدققين الذين ترغب في تشغيلهم"
+        },
+        "folder": {
+            "argument": "--مجلد",
```

I think the "argument" should not be translated?

```suggestion
            "argument": "--folder",
```